lexicon/modification.py

```diff
@@ -178,7 +184,12 @@ def run(self):
                 for lemma in lex.lemmata:
                     # sort by first orth entry
                     orth_key = lemma.orth[0] if lemma.orth else ""
+                    if self.deduplicate_lemmata:
+                        # don't add the lemma when there's already an equal lemma
+                        if len(lemma_dict[orth_key]) > 0 and lemma == lemma_dict[orth_key][0]:
+                            continue
                     lemma_dict[orth_key].append(lemma)
+            print(lemma_dict)
```

Comment:
Was this just for debugging, or is this intended?
```suggestion
```

Reply:
Just debugging :sweat_smile: